ansible_runner/display_callback/module.py

```diff
@@ -20,6 +20,8 @@
 # Python
 import collections
 import contextlib
+import json
```

Comment:
Are these new imports necessary?

Reply:
No, think this was cruft from changes that used to be here. Removed.